url.bs

```diff
@@ -2531,10 +2529,9 @@ with care to prevent spoofing:
  <li><p>Browsers should render a <a for=/>URL</a>'s <a for=url>host</a> using
  <a>domain to Unicode</a>.
 
- <p class="note no-backref">Note that non-ASCII characters can be used in <a
- href="http://unicode.org/faq/idn.html#26">homograph</a> spoofing attacks. Consider detecting <a
- href="http://www.unicode.org/reports/tr39/#Confusable_Detection">confusable characters</a> or
- warning when they are in use.
+ <p class="note no-backref">Note that various characters can be used in homograph [[IDNFAQ]]
+ spoofing attacks. Consider detecting confusable characters [[UTR39]] and warning when they are in
```

Comment:
The reference here should now be to `[[UTS39]]` instead of `[[UTR39]]`. (I should’ve added it to Specref as  UTS39 to begin with, but anyway https://github.com/tobie/specref/pull/522 from @TimothyGu subsequently revamped all the Specref handling for Unicode references, including that fix.)

Reply:
Fixed, thanks!